README.md

```diff
@@ -38,6 +38,8 @@ $ s3-object-router \
 Usage of s3-object-router:
   -bucket string
     	destination S3 bucket name
+  -convert-json-record
+        convert the record to JSON format. use if -parser when not "json"
```

Comment:
`-format (json|none) (default none)` is better I think. For example, enable to add `-format parquet` in feature.